corehq/apps/app_manager/tests/test_case_list_form.py

```diff
@@ -153,6 +163,14 @@ def test_case_list_form_parent_child_basic(self):
         )
 
         factory.form_requires_case(update_person_form, parent_case_type='house')
+        register_person_module.case_details.short.columns.append(
+            DetailColumn(
+                header={'en': 'a'},
+                model='case',
+                field='house/case_name',  # Include a parent case property in the case list
```

Comment:
Although this is technically legal I'd prefer if we stuck to the normal references and made this `parent/case_name`